.github/scripts/install_singularity.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -eo pipefail
+
+sudo apt-get update
+DEBIAN_FRONTEND=noninteractive sudo apt-get install -y \
+    build-essential \
+    libssl-dev \
+    uuid-dev \
+    libgpgme11-dev \
+    squashfs-tools \
+    libseccomp-dev \
+    wget \
+    pkg-config \
+    git \
+    cryptsetup
+
+echo $(which go)
```

Comment:
Are go-related lines needed in this file?

Reply:
just check that go compiler is installed.